print/nuxt.config.js

```diff
@@ -58,8 +57,19 @@ export default {
   modules: [
     // Doc: https://axios.nuxtjs.org/usage
     '@nuxtjs/axios',
+
     // Doc: https://sentry.nuxtjs.org/guide/usage
     '@nuxtjs/sentry',
+
+    // Doc: https://i18n.nuxtjs.org/basic-usage
+    [
+      '@nuxtjs/i18n',
+      {
+        locales: ['en'], // only 'en' to avoid that Nuxt generates URLs for each locale. We only need the vueI18n library directly
```

Comment:
```suggestion
        locales: ['en'], // only 'en' to avoid that Nuxt generates URLs for each locale. We only need the @nustjs/i18n library directly
```

Reply:
*nuxtjs

Reply:
I actually really meant the vue-i18n library: We don't really need the advanced features of @nuxtjs/i18n. But @nuxtjs/i18n helps to install a proper/memory-leakage free setup for vue-i18n. I have slightly adjusted the comment to clarify:
https://github.com/ecamp/ecamp3/pull/3187/commits/dc3fcf3941206f91e4facbd6f97a243cd2ac3f1c